api/v1alpha2/systemconfiguration_types.go

```diff
@@ -113,3 +104,13 @@ type SystemConfigurationList struct {
 func init() {
 	SchemeBuilder.Register(&SystemConfiguration{}, &SystemConfigurationList{})
 }
+
+func (in *SystemConfiguration) Computes() []string {
+	computes := make([]string, 0)
```

Comment:
If a cluster is going to have 11,000 computes, then I should re-do this.  Walk the loop once to count the computes and pre-allocate the whole array, then walk again to populate the array.